### ISSUE TITLE
TTS inference and vocoding issues

### ISSUE THREAD

**Issue by hsmallbone** [NVIDIA/NeMo#701]
Hello, I am finding this library has nice and stable TTS training. However, when I use tts_infer.py with my trained tacotron2 model (finetuned from the pretrained model), the output wav is quite noisy without using the denoiser, and the waveglow vocoder is slow and requires a GPU. I was wondering:
1. if this level of noise is expected when using the waveglow vocoder? Should I be finetuning waveglow or always using a certain denoising level? 
2. when it will be possible to infer using fastspeech or use a different faster vocoder such as ParallelWaveGAN?

I have tried to feed the mel spectrogram output from tacotron2 into a pretrained ParallelWaveGAN from another project (ESPNet) but the output ends up garbage unless I scale the spectrograms and then it sounds robotic.

**Comment by MuyangDu** [comment]
Not sure if you are talking about the line noise of WaveGlow, this is still an open issue: https://github.com/NVIDIA/waveglow/issues/55

**Comment by hsmallbone** [comment]
@MuyangDu thanks for the link, sounds like the best answer for that is to either use the denoiser, try different waveglow sigmas with the pretrained waveglow model or retrain waveglow? That's why I was trying to feed the predicted mel spectrograms into another vocoder but it didn't work too well.

**Comment by MuyangDu** [comment]
Yes, you can try to retrain WaveGlow on you own dataset and try to adjust the sigma to get better performance. The audio preprocessing in NeMo is different from ESPNet (for example, whether do normalization or scale to spectrogram, hop size,win size in stft, etc...). That's why you can't directly feed the spectrogram output by NeMo to ESPNet's vocoder.

**Comment by hsmallbone** [comment]
I guess I will wait for a fastspeech / different vocoder implementation then.

**Comment by hsmallbone** [comment]
@MuyangDu assuming there is no way to adjust for the preprocessing changes?
